### PR TITLE
resolves xero-node 421

### LINKED ISSUE #421: Wrong openid-client types
**SDK you're using (please complete the following information):**
 - Version: 4.5.1

**Describe the bug**
`xero-node` is overriding openid-client's types. So if you use `xero-node` and `openid-client` simultaneously, you get definitions from `xero-node`.

This is causing an issue because in the last types versions (https://github.com/panva/node-openid-client/blob/master/types/index.d.ts), some functions are added that your copy is missing (`requestResource` most notably).

You may argue that you have a version in your package.json, but it's in `"^3.9.2"`, which means if I install `openid-client` aside of `xero-node` for other usage and get the latest version, I get a type conflict.

Either fix `openid-client` version so yarn/node installs the two packages versions or (better) do not copy types like this and leverage typescript's native usage

### PR DESCRIPTION
**Pull request by RettBehrens**
resolves #421

### CODE REVIEW

**Comment by RettBehrens** [comment]
this now also resolves issue #488... just need to update sample app and readme example before merging

File: README.md [hunk 1]
```diff
@@ -315,19 +315,19 @@ await xero.buildConsentUrl()
...
 const tokenSet = await xero.readTokenSet();
-const now = new Date().getTime()
 
-if (tokenSet.expires_in > now) {
+tokenSet.expired() // returns boolean true/false
+tokenSet.expires_in // returns seconds
+tokenSet.expires_at // returns milliseconds
+new Date(tokenSet.expires_at * 1000).toLocaleString()) // readable expiration
+
+if (tokenSet.expired()) {
```
Comment (SerKnight) [inline README.md:324]: Nice adds

**Review by SerKnight — APPROVED** [review]
Nice

**Comment by RettBehrens** [comment]
Resolves #421 by removing `src/untyped.ts` from the project, which was previously overwriting openid-client types for some devs.
Resolves #488 by ensuring all TokenSets are of valid TokenSet type per openid-client TokenSet class. Previously `refreshWithRefreshToken` was not getting a consistent TokenSet given it was coming directly from Xero identity without being passed through openid-client library. Also updated sample app and sdk readme to leverage built-in `tokenSet.expired` method

**Pull request merged**